src/ReservoirPair.sol

```diff
@@ -357,32 +361,32 @@ abstract contract ReservoirPair is ReservoirERC20 {
         assetManager.afterLiquidityEvent();
     }
 
-    function adjustManagement(int256 token0Change, int256 token1Change) external {
+    function adjustManagement(int256 aToken0Change, int256 aToken1Change) external {
         require(msg.sender == address(assetManager), "AMP: AUTH_NOT_MANAGER");
-        require(token0Change != type(int256).min && token1Change != type(int256).min, "AMP: CAST_WOULD_OVERFLOW");
+        require(aToken0Change != type(int256).min && aToken1Change != type(int256).min, "AMP: CAST_WOULD_OVERFLOW");
 
-        if (token0Change > 0) {
-            uint104 lDelta = uint104(uint256(int256(token0Change)));
+        if (aToken0Change > 0) {
+            uint104 lDelta = uint104(uint256(aToken0Change));
             token0Managed += lDelta;
             token0.transfer(msg.sender, lDelta);
-        } else if (token0Change < 0) {
-            uint104 lDelta = uint104(uint256(int256(-token0Change)));
+        } else if (aToken0Change < 0) {
+            uint104 lDelta = uint104(uint256(-aToken0Change));
 
             // solhint-disable-next-line reentrancy
             token0Managed -= lDelta;
 
             token0.transferFrom(msg.sender, address(this), lDelta);
```

Comment:
Should we be using safeTransferFrom everywhere?

Reply:
good thinking... I'll prolly use solmate's impl

Reply:
implemented